libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/map/NavigationMapboxMap.java

```diff
@@ -82,20 +83,23 @@
   private LocationFpsDelegate locationFpsDelegate;
 
   /**
-   * Constructor that can be used once {@link com.mapbox.mapboxsdk.maps.OnMapReadyCallback}
+   * Constructor that can be used once {@link OnMapReadyCallback}
    * has been called via {@link MapView#getMapAsync(OnMapReadyCallback)}.
    *
-   * @param mapView   for map size and Context
-   * @param mapboxMap for APIs to interact with the map
+   * @param mapView           for map size and Context
+   * @param mapboxMap         for APIs to interact with the map
+   * @param routeBelowLayerId optionally pass in a layer id to place the route line below
    */
-  public NavigationMapboxMap(@NonNull MapView mapView, @NonNull MapboxMap mapboxMap) {
+  public NavigationMapboxMap(@NonNull MapView mapView,
+                             @NonNull MapboxMap mapboxMap,
+                             @Nullable String routeBelowLayerId) {
```

Comment:
Let's expose one more constructor where `null` for `routeBelowLayerId` is the default option.